### PR TITLE
Remove Hub incomplete/complete and guidance

### PR DESCRIPTION
**Pull request by LJBabbage**
### What is the context of this PR?

With guidance removed from the hub, complete and incomplete becomes redundant and title can be moved to submission

The validator branch which goes with this is add-warning-to-hub-and-rationalise

### How to review
Make sure the keys have been removed/moved as expected

### Checklist

- [ ] Jsonnet files conform to the latest [style guide](/ONSdigital/eq-questionnaire-schemas/blob/master/style_guide.md)

### CODE REVIEW

**Comment by LJBabbage** [comment]
> Is it worth adding the guidance/warning now while we are here?. Happy even if separate.

We could of easily, I will pick it up straight after

**Comment by JamesGardiner** [comment]
I've approved based on this validating once your validator pr merges

File: source/jsonnet/england-wales/ccs_household.jsonnet [hunk 1]
```diff
@@ -96,15 +96,9 @@ function(region_code, census_month_year_date) {
...
-    complete: {
-      title: 'Submit census',
-      guidance: 'Please submit this census to complete it',
-    },
-    incomplete: {
-      guidance: 'You must complete all sections in order to complete this census',
-    },
     submission: {
       button: 'Submit census',
+      title: 'Submit census',
```
Comment (ajmaddaford) [inline source/jsonnet/england-wales/ccs_household.jsonnet:101]: The `submission` property is now at the wrong level in these schemas.

**Comment by MebinAbraham** [comment]
Translation package will need updating once released. Happy for it to be a separate PR.

**Pull request merged**